### ISSUE TITLE
Container sandbox in indeterminate state - can't be started or stopped.

### ISSUE THREAD

**Issue by coozy** [docker/docker#24825]
systemctl restart docker  - docker didn't go down gracefully
one of the containers is in a state where it can't be started or stopped:
```
root@ip-172-31-28-41 porx]# docker ps 
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[root@ip-172-31-28-41 porx]# docker ps -a
CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS                     PORTS               NAMES
149724c0af84        coozy/passd   "/docker-entry-point."   7 hours ago         Exited (128) 6 hours ago                      admiring_hugle
[root@ip-172-31-28-41 porx]# docker start 149724c0af84
Error response from daemon: Container 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571 is aleady active
Error: failed to start containers: 149724c0af84
```
Logs:

```

Jul 19 19:28:19 ip-172-31-28-41.us-west-1.compute.internal systemd[1]: docker.service stop-sigterm timed out. Killing.
Jul 19 19:28:19 ip-172-31-28-41.us-west-1.compute.internal systemd[1]: Unit docker.service entered failed state.Jul 19 19:28:19 ip-172-31-28-41.us-west-1.compute.internal systemd[1]: docker.service failed.
Jul 19 19:28:19 ip-172-31-28-41.us-west-1.compute.internal systemd[1]: Starting Docker Application Container Engine...Jul 19 19:28:19 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:19.263265834Z" level=info msg="New containerd process, pid: 5441\n"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.326262740Z" level=warning msg="devmapper: Usage of loopback devices is strongly discouraged for production use. Please use `--storage-opt dm.thinpooldev` or use `man docker` to refer to dm.thinpooldev section."
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.365734971Z" level=warning msg="devmapper: Base device already exists and has filesystem xfs on it. User specified filesystem  will be ignored."
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.394067279Z" level=info msg="[graphdriver] using prior storage driver \"devicemapper\""
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.449721878Z" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.476023593Z" level=info msg="Firewalld running: false"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.551758532Z" level=info msg="Removing stale sandbox cfbefe5092bd9cb2edc4db79e778d8c708d375ef289d61365f17fb2e225e4060 (149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571)"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.565812135Z" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.599408193Z" level=warning msg="mountpoint for pids not found"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.599661571Z" level=info msg="Loading containers: start."
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: .time="2016-07-19T19:28:20.599839444Z" level=error msg="Failed to load container 133264a27bc3020cb59b1d06a02a26265598859939548947c622bd3864406f3b: open /var/lib/docker/containers/133264a27bc3020cb59b1d06a02a26265598859939548947c622bd3864406f3b/config.v2.json: no such file or directory"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: ..time="2016-07-19T19:28:20.604097436Z" level=warning msg="error locating sandbox id a350989c0fac8b2e5687a158dce96306781d59945d0ab72f3f4ab2525951a8dd: sandbox a350989c0fac8b2e5687a158dce96306781d59945d0ab72f3f4ab2525951a8dd not found"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.604148819Z" level=warning msg="failed to cleanup ipc mounts:\nfailed to umount /var/lib/docker/containers/4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d/shm: invalid argument"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.604181486Z" level=error msg="devmapper: Error unmounting device d4a63af289d558d90afdfed9142fc204f530f4b56c04b82168e1bb2dce3f91f9: invalid argument"
Jul 19 19:28:20 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:20.604196969Z" level=error msg="Error unmounting container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d: invalid argument"
Jul 19 19:28:30 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:30.606853033Z" level=error msg="error sending sigkill to 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571: rpc error: code = 2 desc = \"no such process\""
Jul 19 19:28:32 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:32.607152348Z" level=warning msg="error locating sandbox id cfbefe5092bd9cb2edc4db79e778d8c708d375ef289d61365f17fb2e225e4060: sandbox cfbefe5092bd9cb2edc4db79e778d8c708d375ef289d61365f17fb2e225e4060 not found"
Jul 19 19:28:32 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:32.862124608Z" level=error msg="Failed to start container 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571: Container 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571 is aleady active"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.864276350Z" level=info msg="Loading containers: done."
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.864294273Z" level=info msg="Daemon has completed initialization"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.864309640Z" level=info msg="Docker daemon" commit=5604cbe graphdriver=devicemapper version=1.11.1
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal systemd[1]: Started Docker Application Container Engine.
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.870969883Z" level=info msg="API listen on /var/run/docker.sock"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.872191649Z" level=error msg="Handler for POST /v1.23/containers/4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d/kill returned error: Cannot kill container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d: Container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d is not running"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.873498550Z" level=error msg="Handler for POST /v1.23/containers/4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d/kill returned error: Cannot kill container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d: Container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d is not running"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.879447209Z" level=error msg="Handler for POST /v1.23/containers/4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d/kill returned error: Cannot kill container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d: Container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d is not running"
Jul 19 19:28:47 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:28:47.882094986Z" level=error msg="Handler for POST /v1.23/containers/4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d/kill returned error: Cannot kill container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d: Container 4cf05f873430341d63340b44fce8c2131d13d3d842785378ec170975d956f71d is not running"
Jul 19 19:31:12 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:31:12.334900901Z" level=error msg="Handler for POST /v1.23/containers/admiring_hugle/restart returned error: Cannot restart container admiring_hugle: Container 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571 is aleady active"
Jul 19 19:31:33 ip-172-31-28-41.us-west-1.compute.internal docker[5435]: time="2016-07-19T19:31:33.705110192Z" level=error msg="Handler for POST /v1.23/containers/admiring_hugle/start returned error: Container 149724c0af84231d8db95363babc382a1e7d2d572b3de611e43bb8ed6c966571 is aleady active"
```

Contents of var/lib/docker/container

```
root@ip-172-31-28-41 porx]# ls -l /var/lib/docker/containers/133264a27bc3020cb59b1d06a02a26265598859939548947c622bd3864406f3b/
total 0

```

**Output of `docker version`:**

```
Client:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Wed Apr 27 00:34:42 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Wed Apr 27 00:34:42 2016
 OS/Arch:      linux/amd64

```

**Output of `docker info`:**

```
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 2
Server Version: 1.11.1
Storage Driver: devicemapper
 Pool Name: docker-202:1-58763008-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 1.623 GB
 Data Space Total: 107.4 GB
 Data Space Available: 24.63 GB
 Metadata Space Used: 2.535 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.145 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.107-RHEL7 (2015-12-01)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: host bridge null
Kernel Version: 3.10.0-327.13.1.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 3.451 GiB
Name: ip-172-31-28-41.us-west-1.compute.internal
ID: 6C3C:2TUA:J7XZ:3VQ4:N436:2CNE:OT4L:QHSK:VMFU:E2OJ:ENU7:6U3D
Docker Root Dir: /var/lib/docker

```

**Comment by abronan** [comment]
Hi @coozy, I think this is related to https://github.com/docker/docker/issues/22851 and a fix was included for docker `1.12`.

/cc @thaJeztah

**Comment by cpuguy83** [comment]
Also 1.11.2

Closing since this should already be fixed.
Please upgrade to 1.11.2 and let us know if you still have a problem.

Thanks!
